accounts/templates/accounts/subnav_content.html

```diff
@@ -16,6 +16,9 @@
 <li class={{ subnav_sync_delicious }}>
```

Comment:
would you mind commenting out the delicious stuff so it doesn't get seen anymore? Since this doesn't do anything right now I'd prefer it wasn't visible.